docker/STAR/1.0.0-alpha/sort_star_input.py

```diff
@@ -0,0 +1,153 @@
+"""Sort fastq pairs and read group information prior to being passed to STAR.
+
+We discovered that when providing STAR with read group split fastqs and also
+supplying STAR with header information via the `--outSAMattrRGline` parameter,
+that the read group information for individual reads becomes jumbled.
+
+The simplest solution for this was to first pass exactly what would be passed
+to STAR into this program, sort it correctly, and write the reordered
+arguments out to files which can be supplied to STAR via the `cat` command.
+"""
+
+import argparse
+import os
+from typing import List, Tuple
+
+
+def sort_lists(
+    target_list: List[any],
```

Comment:
```suggestion
    target_list: List[Any],
```
You would also need to import `Any` from `typing`.

or, if you know the types that you expect
```suggestion
    target_list: List[Union[int, str]],
```